Jellyfin.Plugin.ServerWMC/WMCService.cs

```diff
@@ -16,6 +16,7 @@
 using MediaBrowser.Controller.Drawing;
 using MediaBrowser.Model.Dto;
 using System.Net;
+using System.Net.Http;
```

Comment:
Is this plugin missing the package reference, or not using `IHttpClientFactory`?
```xml
<PackageReference Include="Microsoft.Extensions.Http" Version="5.0.0" />
```

Reply:
I've removed all `using` directives. Looks like this plugin communicates of websocket so `HttpClientFactory`/`IHttpClient` were never used